apps/Sphero SDK/install

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+rm -rf ~/sphero-sdk-raspberrypi-python.git
+
+git_clone https://github.com/sphero-inc/sphero-sdk-raspberrypi-python.git || error "Failed to clone repository!"
+cd ~/sphero-sdk-raspberrypi-python
+
+sudo python3 setup.py install || error "Failed to run setup.py!"
+sudo ./first-time-setup.sh || error "Failed to run first time setup!"
```

Comment:
Hi, I'm the primary maintainer of the Sphero RVR SDK.  You don't need sudo for `first-time-setup.sh`, and there's no need to run setup.py at all (it's left over from an earlier effort and I should really remove it).

For reference, [here is the instructions page](https://sdk.sphero.com/docs/getting_started/raspberry_pi/raspberry_pi_setup_advanced) for installation.

Thanks for adding our SDK into pi-apps!

Reply:
I understand. Thanks for taking the time to reply!